modules/azure-ext/pom.xml

```diff
@@ -32,7 +32,7 @@
     <url>https://ignite.apache.org</url>
 
     <properties>
-        <spring.version>5.3.8</spring.version>
+        <spring.version>5.3.19</spring.version>
```

Comment:
I would suggest renaming all the properties related to Spring 5.3 version - `spring53.version` since we already have a spring versions in the parent pom an it may lead to the so-called dependency-hell case.

Reply:
Done (I hope I understood your suggestion correctly)